### PR TITLE
feat: update project info and node list

### PR DESCRIPTION
**Pull request by dellkeji**

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-bcs-saas?pullRequest=98) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-bcs-saas?pullRequest=98) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-bcs-saas?pullRequest=98) it.</sub>

File: bcs-app/backend/apps/cluster/views/node.py [hunk 1]
```diff
@@ -465,7 +465,8 @@ def update_agent_status(self, cc_host_map, gse_host_status):
                     continue
                 item_info = gse_host_status_map[item]
                 item_exist = item_info.get('exist') or item_info.get('bk_agent_alive')
-                exist = exist if exist > 0 else item_exist
+                # 防止出现None情况
+                exist = exist if exist > 0 else (item_exist or exist)
```
Comment (jamesgetx) [inline bcs-app/backend/apps/cluster/views/node.py]: exist的含义是？取值范围是？如果是表示是否存在，那取值0或者1？

Comment (dellkeji) [reply bcs-app/backend/apps/cluster/views/node.py]: exsit这个有None的情况

Comment (ifooth) [reply bcs-app/backend/apps/cluster/views/node.py]:
```
def update_agent_status(self, cc_host_map, gse_host_status):
    gse_host_status_map = {info['ip']: info for info in gse_host_status}

    def is_agent_exist(ips):
        # one host may has many eth(ip)
        ip_list = ips.split(',')
        for item in ip_list:
            if item not in gse_host_status:
                continue
            item_info = gse_host_status_map[item]
            item_exist = item_info.get('exist') or item_info.get('bk_agent_alive')
            if item_exist:
                return True
                
        return False

        # render agent status
        for ips in cc_host_map:
            cc_host_map[ips]['agent'] = self.is_agent_exist(ips)
```

**Pull request merged**